sdcm/utils/raft/common.py

```diff
@@ -44,6 +50,29 @@ def validate_raft_on_nodes(nodes: list[BaseNode]) -> None:
     LOGGER.debug("Raft is ready!")
 
 
+def get_topology_coordinator_node(cluster: BaseScyllaCluster) -> BaseNode | None:
+    active_nodes: list[BaseNode] = cluster.get_nodes_up_and_normal()
+    if not active_nodes[0].raft.is_cluster_topology_consistent():
+        return
```

Comment:
wasn't @temichus intention to avoid such if statements? Shouldn't it go under raft object?